src/domain/opendatafabric/src/identity/dataset_identity.rs

```diff
@@ -388,7 +427,7 @@ impl_serde!(DatasetAlias, DatasetAliasSerdeVisitor);
 
 ////////////////////////////////////////////////////////////////////////////////
 
-#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Debug, Clone, Eq, PartialOrd, Ord)]
```

Comment:
Same here, I wonder if `PartialOrd` is necessary to be customized too.

Reply:
Due to lastest changes I moved custom `PartialEq ` implementation to `newtype_istr` level